docs: Fix unocss readme (#1329)
wxt-dev/wxt: packages/unocss/README.md

export default defineConfig({
  modules: ['@wxt-dev/unocss'],
  unocss: {

>     // Exclude unocss apply for popup/main.ts and background
>     excludeEntrypoints: ['popup/main.ts', 'background'],

Reviewer: ```suggestion
    // Exclude unocss from running for the background
    excludeEntrypoints: ['background'],
```
